app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,14 @@
+.top {
+  background-image: url("https://images.unsplash.com/photo-1429305336325-b84ace7eba3b?q=80&fm=jpg&s=8cf84b970497806cc96a06d6aba2b86b");
```

Comment:
URL `https://images.unsplash.com/photo-1429305336325-b84ace7eba3b?q=80&fm=jpg&s=8cf84b970497806cc96a06d6aba2b86b` should not contain protocol or domain